### PR TITLE
Fix the issue that WIFI update does not work for ESP32 WIFI target.

### PR DESCRIPTION
**Pull request by qubic**
Add WIFI target for DIY 2400 TX.

### CODE REVIEW

File: src/python/build_flags.py [hunk 1]
```diff
@@ -153,3 +153,9 @@ def get_git_sha():
 if stm:
     env['UPLOAD_PROTOCOL'] = 'custom'
 
+# Set upload_protovol = 'custom' for ESP32 MCUs WIFI target
```
Comment (pkendall64) [inline src/python/build_flags.py:156]: These changes are not necessary if the change above is made

File: src/python/build_env_setup.py [hunk 2]
```diff
@@ -49,3 +49,5 @@
     if "_WIFI" in target_name:
         env.Replace(UPLOAD_PROTOCOL="custom")
         env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
+elif platform in ['espressif32'] and "_WIFI" in target_name:
+    env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
```
Comment (pkendall64) [inline src/python/build_env_setup.py:53]: Add add `env.Replace(UPLOAD_PROTOCOL="custom")` above this line, so the `build_flags.py` changes are not necessary.

Comment (qubic) [reply src/python/build_env_setup.py:53]: It's necessary, i have add env.Replace(UPLOAD_PROTOCOL="custom") above this line, it does not work, it still use esptool

Comment (pkendall64) [reply src/python/build_env_setup.py:53]: I have change this to section to
```
elif platform in ['espressif32'] and "_WIFI" in target_name:
    env.Replace(UPLOAD_PROTOCOL="custom")
    env.Replace(UPLOADCMD=upload_via_esp8266_backpack.on_upload)
```

and add
```
upload_port = 10.0.0.1
```
 to the end of the target definition in the diy_2400.ini file.i.e.
```
[env:DIY_2400_TX_ESP32_SX1280_E28_via_WIFI]
extends = env:DIY_2400_TX_ESP32_SX1280_E28_via_UART
upload_port = 10.0.0.1
```

and it all works.  I don't this build_flags is the right place to mucking about with upload stuff. That should be for building only.

Comment (qubic) [reply src/python/build_env_setup.py:53]: This does not work for esp backpack which using elrs_tx.local instead of 10.0.0.1

Comment (pkendall64) [reply src/python/build_env_setup.py:53]: For the DIY 2400 the backpack hardware is unused and you will be flashing the ESP32 directly over wifi.
You shouldn't have anything on the backpack ESP8266 on those systems yet as the only backpack code is teh one used to flash the STM32 based systems.

Comment (qubic) [reply src/python/build_env_setup.py:53]: You are right, have you made another pr with these code.

Comment (pkendall64) [reply src/python/build_env_setup.py:53]: If you update your code and push it will update the PR automatically.

**Review by pkendall64 — APPROVED** [review]

**Pull request merged**